app/assets/javascripts/language_selector.js

```diff
@@ -0,0 +1,6 @@
+$(document).ready(function () {
+  $(".language-change-trigger").on("change", function () {
```

Comment:
In the interests of ensuring turbo compatibility it's better to write this as:

```
$(document).on("change", ".language-change-trigger", function () {
```

it probably doesn't natter in this case as it's in the header but there's no reason not to be safe in case it ever gets moved.

Reply:
Code was updated according to the recommendation.